tools/cleed_io/slicer.py

```
@@ -0,0 +1,134 @@
+import numpy as np
+from ase import Atoms
+from ase.geometry import cell_to_cellpar
+
+def align_surface(atoms: Atoms, vacuum_direction='z', tolerance=1e-3) -> Atoms:
```

Ruff (0.14.8)

Unused function argument: `vacuum_direction`

(ARG001)